aws_lambda_builders/workflows/rust_cargo/actions.py

```diff
@@ -0,0 +1,148 @@
+"""
+Rust Cargo build actions
+"""
+
+import logging
+import os
+
+from aws_lambda_builders.workflow import BuildMode
+from aws_lambda_builders.actions import ActionFailedError, BaseAction, Purpose
+from aws_lambda_builders.architecture import X86_64, ARM64
+from .exceptions import CargoLambdaExecutionException
+from .utils import OSUtils
+
+
+LOG = logging.getLogger(__name__)
+
+
+class RustCargoLambdaBuildAction(BaseAction):
+    NAME = "CargoLambdaBuild"
+    DESCRIPTION = "Building the project using Cargo Lambda"
+    PURPOSE = Purpose.COMPILE_SOURCE
+
+    def __init__(
+        self,
+        source_dir,
+        binaries,
+        mode,
+        subprocess_cargo_lambda,
+        architecture=X86_64,
+        handler=None,
+        flags=None,
+    ):
+        """
+        Build the a Rust executable
+
+        :type source_dir: str
```

Comment:
nit: can you also update doc formats to numpy format (https://www.dataquest.io/blog/documenting-in-python-with-docstrings/)

We are not doing a good job to align on that in this library, but at least we are trying to address them with new additions.